web/massastation/src/pages/Index/Index.tsx

```diff
@@ -22,19 +22,39 @@ import {
   IMassaPlugin,
   IMassaStore,
 } from '../../../../shared/interfaces/IPlugin';
-import { usePost, useResource } from '../../custom/api';
 import Intl from '../../i18n/i18n';
 import { routeFor } from '../../utils';
 import { useConfigStore } from '../../store/store';
+import { usePost, useResource } from '../../custom/api';
+import { UseQueryResult } from '@tanstack/react-query';
 
 export function Index() {
+  const plugins = useResource<IMassaPlugin[]>('plugin-manager');
+  const store = useResource<IMassaStore[]>('plugin-store');
+  const { isLoading: isPluginsLoading } = plugins;
+  const { isLoading: isStoreLoading } = store;
+
+  return isPluginsLoading || isStoreLoading ? (
+    <>Loading</>
```

Comment:
could you put this in translation keys?

Reply:
should I use the I18n on this pr  before checking out the loading with the skeleton on another one ?

Reply:
...or if we don't use just set `null`

Reply:
another PR